advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx

```diff
@@ -0,0 +1,321 @@
+---
+title: 'Using'
+description: 'Walkthrough of example usage scenarios'
+---
+
+The following steps demonstrate Precisely Connect CDC’s Replication functionality with EDB Postgres Advanced Server.
+
+## Configure EDB Postgres Advanced Server as Source Instance in Connect CDC Director
+
+The following steps demonstrate the setup and configuration of EDB Postgres Advanced Server as source instance for Connect CDC Director. 
+
+1. Open Connect CDC Director.
+
+2. Click New option.
+
+3. Right click Hosts, Servers, Tables and select New Host.  
+
+4. Enter information for the Host, Host is the machine where Connect CDC Director is running.
+
+![Create New Host](Images/newhost.png)
+
+5. Right Click on the newly added Host and select Test Connection, a success message will be displayed.
+
+![Test Connection with New Host](Images/tstconnection.png)
+
+6. Right Click on the newly added Host and select New Server.
+
+7. Enter Connection Information for EDB Postgres Advanced Server as Source Instance (Server name is where the database name is given).
+
+![Create New Server](Images/newserver.png)
+
+8. Right Click on the newly added Server and select Install Source Metabase.
+
+9. Provide EDB Postgres Advanced Server Source Instance User and Password on User ID and Password Screen.
+
+![User ID and Password Screen](Images/idpassword.png)
+
+10. Provide password of rpuser (refer to the user name and password given on New Server Screen) when prompted on the User ID and Password screen.
+
+![User ID and Password Screen](Images/idpassword1.png)
+
+11. Click Ok on Install Source Metabase screen.
+
+![Install Source Metabase](Images/installmetabase.png)
```

Comment:
I think we can just use this:

<img width="705" alt="InstallSourceMetabaseOKScreen" src="https://github.com/EnterpriseDB/docs/assets/106532490/0fbac7ea-9e33-4df0-98be-e7b6726d1644">

Reply:
done